docs/site/tutorials/connectors/Mongodb-tutorial.md

```diff
@@ -0,0 +1,213 @@
+setups:
+
+```bash
+$ lb4 datasource
+? Datasource name: db
+? Select the connector for db:
+  ...
+  Redis key-value connector (supported by StrongLoop)
+❯ MongoDB (supported by StrongLoop)
+  MySQL (supported by StrongLoop)
+  ...
+? Connection String url to override other settings (eg: mongodb://username:passw
+ord@hostname:port/database):
+? host: localhost
+? port: 27017
+? user:
+? password: [hidden]
+? database: demo
+? Feature supported by MongoDB v3.1.0 and above: Yes
+
+Datasource Db was created in src/datasources/
+```
+
+Under `src/datasources/db.datasource.ts`, we can find the `DBDataSource` class
+and the config we just set:
+
+```ts
+const config = {
+  name: 'db',
+  connector: 'mongodb',
+  url: '',
+  host: 'localhost',
+  port: 27017,
+  user: '',
+  password: '',
+  database: 'demo',
+};
+```
+
+{% include important.html content="please make sure you are using `loopback-connector-mongodb` package version 5.2.1
+or above to handle `ObjectId` properly." %}
+
+### 4. Create repositories
+
+[Repository](../../Repository.md) is an artifact that ties the model and the
+datasource. We will need to create the repository for the `User` class to
+accessing the database. The steps of creating `UserRepository` by running
+`lb4 repository` are skipped here:
+
+```ts
+// imports
+export class UserRepository extends DefaultCrudRepository<
+  User,
+  typeof User.prototype.id,
+  UserRelations
+> {
+  constructor(@inject('datasources.db') dataSource: DbDataSource) {
+    super(User, dataSource);
+  }
+}
+```
+
+### 5. Create endpoints and view data using API Explorer
+
+Once we built a [controller](../../Controllers.md) with `lb4 controller` to
+handle requests, from the project root, start the app:
+
+```bash
+$ npm start
+```
+
+We can verify what we just created with API Explorer
+`http://localhost:3000/explorer/`.
+
+## Data Mapping Properties
+
+If you'd like to use different names for the collection/fields and the
+model/properties, it can be achieved by configuring the model
+definition/property definition. Take the `User` model as an example, if we'd
+like to name the collection as `MY_USER` in the database and also use uppercase
+for properties `name` and `hasAccount`, the following settings would allow so:
+
+```ts
+@model({
+  settings: {
+    mongodb: {collection: 'MY_USER'},
+  },
+})
+export class User extends Entity {
+  @property({
+    type: 'string',
+    id: true,
+    generated: true,
+  })
+  id: string;
+
+  @property({
+    type: 'string',
+    mongodb: {
+      fieldName: 'NAME',
+    },
+  })
+  name?: string;
+
+  @property({
+    type: 'boolean',
+    required: true,
+    mongodb: {
+      fieldName: 'HASACCOUNT',
+    },
+  })
+  hasAccount: boolean;
+}
+```
+
+{% include important.html content="Since in MongoDB `_id` is reserved for the primary key, LoopBack DOES NOT allow to customize the field name for the id property. Please use `id` as is." %}
```

Comment:
maybe highlight using bold style instead of capitalizing the letters :) 

```suggestion
{% include important.html content="Since in MongoDB `_id` is reserved for the primary key, LoopBack **does not** allow to customize the field name for the id property. Please use `id` as is." %}
```